mozilla-release/browser/components/nsBrowserGlue.js

```diff
@@ -1236,7 +1236,8 @@ BrowserGlue.prototype = {
     // There are several cases where we won't show a dialog here:
     // 1. There is only 1 tab open in 1 window
     // 2. The session will be restored at startup, indicated by
-    //    browser.startup.page == 3 or browser.sessionstore.resume_session_once == true
+    //    browser.startup.restoreTabs == true or
```

Comment:
Guess it could be useful to document this somewhere outside of source code. More advanced CLIQZ users may be confused with changes in firefox prefs behaviour.
Documentation for our own prefs can also be useful.